exchange/exchange.go

```diff
@@ -474,6 +478,12 @@ func (e *exchange) getAllBids(
 					e.me.RecordAdapterBidReceived(bidderRequest.BidderLabels, bid.bidType, bid.bid.AdM != "")
 				}
 			}
+
+			//put stored req data back to bid.ext, map by bid.imp_id
+			if len(impToStoredReq) != 0 {
+				insertStoredImpData(impToStoredReq, bids)
```

Comment:
I think we missed a key part of the spec. This should only be done if a new config option `imp.ext.prebid.options.echovideoattrs` is set to true.

Reply:
Wow, for some reason I missed this part. Good catch! Check added.